spec/controllers/state_file/questions/w2_controller_spec.rb

```diff
@@ -111,13 +111,38 @@
           }
         end
 
-        it "updates Box 14 fields" do
+        it "updates MD Box 14 fields" do
           post :update, params: params
           state_file_w2.reload
           expect(state_file_w2.box14_stpickup).to eq 230
         end
       end
 
+      context "with NJ Box 14 fields" do
```

Comment:
[boulder] hm, i guess we didn't do this for `box14_stpickup` but should we add validations for these fields so there's a sensible error when they enter an invalid value?

Reply:
Added numericality validations for all four box 14 codes